dnf-behave-tests/features/alias-command.feature

```diff
@@ -3,10 +3,7 @@
 Feature: Test for alias command
 
 Background:
-  Given I delete directory "/etc/dnf/aliases.d/"
-    And I delete file "/etc/yum.repos.d/*.repo" with globs
-    And I delete directory "/var/lib/dnf/modulefailsafe/"
-    And I use repository "alias-command"
```

Comment:
You're removing similar lines from many places.
Is it safe? I did not go deep in the code review, that's why I'm asking even if it's maybe obvious.

Reply:
What this does is cleans up the environment (the host system, since it's for `@no_installroot`) for the tests. I've moved this preparation to `environment.py` (it's in this commit), where I move these files to a backup location (so that if this gets run on a host system on accident, the user doesn't lose the files).

So yeah I think it's safe and shares code that always needs to be executed when running `@no_installroot`.

Reply:
FTR it may be a bit easier to run the "cleaning" code in `environment.py` on a host system by accident, but hence it does a backup - I still think it's the best way as opposed to having to run all the steps in each destructive test (and forgetting to do that).